Sources/Extensions/UIButton+ImageDownloader.swift

```diff
@@ -9,11 +9,25 @@
 import UIKit
 
 public extension ObjectWrapper where T: UIButton {
```

Comment:
Should this be supported? With a button I could have multiple images one for each state and this doesn't work with that. maybe it should be removed? do you use it?

Reply:
The setImage function does take a state argument and can properly set different images.

The imageUrl property is a convenience that will set the normal state by default.

Let me know if you think we should remove the imageUrl convenience property.